Don't use -mfpmath=sse -msse2 on Mac OS universal ARM/Intel, PowerPC/Intel
builds.
https://github.com/silnrsi/graphite/pull/84
--- src/CMakeLists.txt.orig	2020-04-01 04:53:13.000000000 +0200
+++ src/CMakeLists.txt	2023-04-04 15:43:50.000000000 +0200
@@ -139,7 +139,7 @@
         COMPILE_FLAGS   "-Wall -Wextra -Wno-unknown-pragmas -Wimplicit-fallthrough -Wendif-labels -Wshadow -Wno-ctor-dtor-privacy -Wno-non-virtual-dtor -fno-rtti -fno-exceptions -fvisibility=hidden -fvisibility-inlines-hidden"
         LINK_FLAGS      "-nodefaultlibs"
         LINKER_LANGUAGE C)
-    if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86|i.86")
+    if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86|i.86" AND NOT "${CMAKE_OSX_ARCHITECTURES}" MATCHES "arm|ppc")
         add_definitions(-mfpmath=sse -msse2)
     endif()
     target_link_libraries(graphite2 c)
